signup/signup.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width =device-width, initial-scale=1.0">
+  <title>로그인</title>
+  <link rel="stylesheet" href="signup.css">
+</head>
+<body>
+  <div class="container">
+    <form>
```

Comment:
(signin.js에서 리뷰 드린것처럼) form의 기본 기능을 이용해 제출을 구현하면 좋을 것 같습니다. 
아래는 간단한 코드이니 참고 부탁드려요!

```suggestion
    <form class="signin-form" action="/folder" method="post">
      ...
    </form>
```

signin.js
```javascript
// DOM 조회
const formElem = document.querySelector(".signin-form");

// 이벤트 핸들러 함수
function handleSubmit(e) {
  // name 함수에 구현된 로직이 들어갑니다.
  // 아이디/비밀번호가 다른 경우 submit의 기본기능을 무효화하는 코드가 들어가야 합니다. (preventDefault())
}

// 이벤트 등록
formElem.addEventListener("submit", handleSubmit);

```